.github/workflows/build.yml

```diff
@@ -60,21 +60,18 @@ jobs:
         if: runner.os == 'Windows'
         uses: ilammy/setup-nasm@v1
 
-      - name: Install ninja-build tool for aws-lc-fips-sys on Windows
-        if: runner.os == 'Windows'
-        uses: seanmiddleditch/gha-setup-ninja@v6
+      # NOW REMOVED FROM THIS FORK:
```

Comment:
I think updates like this throughout these would read a little better in general:

```diff
-      # NOW REMOVED FROM THIS FORK:
+      # [FEATURE REMOVED FROM THIS FORK]